lib/pf/Switch/Juniper/EX2300.pm

```diff
@@ -46,6 +46,28 @@ Send a Disconnect request to disconnect a mac
 
 =cut
 
+=head2 getVoipVsa
+
+Get Voice over IP RADIUS Vendor Specific Attribute (VSA).
+For now it returns the voiceVlan untagged since Juniper supports multiple untagged VLAN in the same interface
+
+=cut
+
+sub getVoipVsa{
+    my ($self) = @_;
+    my $logger = $self->logger;
+    my $voiceVlan = $self->{'_voiceVlan'};
+    
+    $logger->info("Accepting phone with Access-Accept on voiceVlan $voiceVlan");
+    
+    return (
+        'Tunnel-Medium-Type' => $RADIUS::ETHERNET,
```

Comment:
These artibutes shuld be deleted.
`Tunnel-Medium-Type => $RADIUS::ETHERNET`
`Tunnel-Type => $RADIUS::VLAN`
`Tunnel-Private-Group-ID => "$voiceVlan"`

Reply:
So just keep Juniper-VoIP-Vlan' => "$voiceVlan", right ?
And confirm it works ?

Reply:
Yes, just keep Juniper-VoIP-Vlan' => "$voiceVlan". I have it in working in production envierment. You can read more [here](https://www.juniper.net/documentation/en_US/release-independent/nce/topics/example/nce159-example-aruba-device-profiling.html). With JunOS 18.3 this is the only way to have [802.1X multi-domain authentication](https://www.juniper.net/documentation/en_US/junos/topics/topic-map/802-1x-and-voip-on-switches.html#id-understanding-8021x-and-voip-on-ex-series-switches).